examples/complete/main.tf

```diff
@@ -89,7 +89,7 @@ module "cloudfront" {
   default_cache_behavior = {
     target_origin_id       = "appsync"
     viewer_protocol_policy = "allow-all"
-
+    response_headers_policy_id = "67f7725c-6f97-4210-82d7-5512b31e9d03"
```

Comment:
Add a comment next to this `magic id` saying that this Policy ID was taken from https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/using-managed-response-headers-policies.html

Reply:
@antonbabenko Please check now.